llvm/tools/llvm-debuginfo-analyzer/README.md

```diff
@@ -0,0 +1,171 @@
+# llvm-debuginfo-analyzer
+
+These are the notes collected during the development, review and test.
+They describe limitations, know issues and future work.
+
+### Remove the use of macros in *LVReader.h* that describe the *bumpallocators*.
```

Comment:
Consider monospace / code style (ticks) instead of emphasis (asterisks) for code things like files and functions